docker/Dockerfile.browse

```diff
@@ -0,0 +1,19 @@
+FROM node:4.4.4
+LABEL node-version="4.4.4"
+
+ARG PLUGINS_REPO
+EXPOSE 4200
+WORKDIR /usr/src
+
+# Update npm
+RUN npm install -g npm@3.9.0
+
+# Clone repos
+RUN git clone https://github.com/dcos/dcos-ui.git /usr/src/dcos-ui
```

Comment:
I explain this in the comment below, but in short, I want to ensure this container will stand the test of time on user's computers. The goal for this container is to make it easy for non-technical users to build the UI off a given branch that they can get from a PR.